main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
Hmm but it's tied with version in URI that's API/App version - therefore I made assumption this is application version and that we update that with every single release (in that case it has to be automated). If you are thinking about having separate versioning for db (can be even simple uint ) this we bump only when we introduce some kind of migration then it's OK. Maybe we won't be able to use it also for enforcing max +1 minor upgrade, but for the main thing it would do just fine.